src/read/pe/file.rs

```diff
@@ -21,6 +22,20 @@ pub type PeFile32<'data, R = &'data [u8]> = PeFile<'data, pe::ImageNtHeaders32,
 /// A PE32+ (64-bit) image file.
 pub type PeFile64<'data, R = &'data [u8]> = PeFile<'data, pe::ImageNtHeaders64, R>;
 
+/// Extracted infos about a possible Rich Header
+#[derive(Debug, Clone, Copy)]
+pub struct RichHeaderInfos<'data> {
```

Comment:
I'm not sure whether this should be here or in `src/pe.rs`

Reply:
`src/pe.rs` is only for raw file format definitions.

Please put this rich header parsing in `src/read/pe/rich.rs`. `RichHeaderEntry` should go there too.